### PR TITLE
Preps for v1.5.2 release

### PR DESCRIPTION
**Pull request by edenhill**
And restructured the source tree to avoid pytest/tox to load confluent-kafka from the source directory rather than what it has installed in its virtual environment.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,38 +11,18 @@ passenv =
...
-[testenv:py35]
-deps =
-    {[base]deps}
-    avro-python3
-
-[testenv:py36]
-deps =
-    {[base]deps}
-    avro-python3
+    -rtests/requirements.txt
```
Comment (rnpridgeon) [inline tox.ini:25]: Missing space

**Review by rnpridgeon — APPROVED** [review]
Nit in tox.ini other than that LGTM

File: tox.ini (hunk 1 above)
Comment (edenhill) [reply tox.ini:25]: That's actually required to be without space.

Comment (rnpridgeon) [reply tox.ini:25]: TIL

**Pull request merged**